content/millennium/r4/encounters/encounter.md

```diff
@@ -9,4 +9,56 @@ title: Encounter | R4 API
 
 ## Overview
 
-Placeholder
+The Encounter resource provides admissions or visits during which health care services were provided to a patient. An encounter has a class to distinguish between different health care settings such as inpatient, outpatient, emergency, etc. A patient may have one medical record number with multiple encounter numbers per facility or organization. There is substantial variance between organizations in the definition of an encounter and what events are aggregated together to constitute an encounter.
+
+## Patch
+
+Patch an existing encounter.
+
+    PATCH /Encounter/:id
+
+_Implementation Notes_
+
+* Only operations on the paths listed below are supported.
+
+### Authorization Types
+
+<%= authorization_types(practitioner: true, system: true) %>
+
+### Headers
+
+<%= headers head: {Authorization: '&lt;OAuth2 Bearer Token>', 'Accept': 'application/fhir+json',
+                   'Content-Type': 'application/json-patch+json', 'If-Match': 'W/"&lt;Current version of the Encounter resource>"'} %>
```

Comment:
Eddy added Accept application/fhir+json as a method param to #headers https://github.com/cerner/fhir.cerner.com/commit/761f781e1eb58169b9b02d2530719b468b68a24d#diff-7718c11ed130ac00a9854e0b089f74b4L27

Reply:
Fixed. https://github.com/cerner/fhir.cerner.com/commit/b3be566e03125de4bdc242726080efeaaef55e65#diff-6590262e197180cc09f6ad70a2a01a5bR30

Reply:
this ends up looking weird on the site with multiple pre blocks

let's continue to set all the headers manually for now

Reply:
Readded. https://github.com/cerner/fhir.cerner.com/pull/240/commits/2f088fe9f198e6cb9d48c1be55d513f5eb8c9f72#diff-97d84edc87e609cdbf80d6ad20cbc681R20